crates/bevy_pbr/src/light.rs

```diff
@@ -368,44 +368,30 @@ pub struct Clusters {
 }
 
 impl Clusters {
-    fn new(tile_size: UVec2, screen_size: UVec2, z_slices: u32, near: f32, far: f32) -> Self {
+    fn new(screen_size: UVec2, dimensions: UVec3, near: f32, far: f32) -> Self {
```

Comment:
since new is only called from one place and with dummy values, we could remove all the parameters and the call to update, just initializing dimensions to zero. Then using Default might be cleaner.

Reply:
Yeah that cuts down on complexity. I'm sold